dist/bin/dotd

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env bash
+
+if [ -z "$PROG_HOME" ] ; then
```

Comment:
By multiple entry points I've meant that you had similar scripts that differ only in what class they invoke in the end.